src/data/mod.rs

```diff
@@ -486,3 +488,15 @@ fn handle_readout_window_manager(
         Err(e) => readout_values.push(Readout::new_err(ReadoutKey::WindowManager, e)),
     }
 }
+
+fn handle_readout_gpu(readout_values: &mut Vec<Readout>, general_readout: &GeneralReadout) {
+    match general_readout.gpus() {
+        Ok(gpus) => {
+            for gpu in gpus {
+                readout_values.push(Readout::new(ReadoutKey::GPU, gpu));
+            }
```

Comment:
Would the list of gpus not be too long if they just separated by commas? The name of the one I have is already the longest readout by itself

Reply:
As far as I can tell, at least with my configuration, it looks like no wrapping is happening at all (in general for `macchina`), maybe that's a separate issue to be opened. I can still add the code to join the gpus into one readout if you want though, or what should I do then? Assuming the wrapping isn't just an issue on my end, I would say that's kind of an important one to look at first since the way it is now text is just getting cut off.

Reply:
To be fair looks like neofetch doesn't wrap either? Maybe I'm being a complete noob here, not sure how the printing of the readout works / how hard it would be to have the text wrap to the next line.

Reply:
Yeah does look interesting. For this now though, should I just push the changes to show all gpus on one readout and then the text wrapping is a separate issue or should I look into using that now?

Reply:
Yeah that's what I meant sorry, do you want me to try and implement this for all readouts here or can it be a separate issue